### PR TITLE
Normalized Process Events package - Take 2

### PR DESCRIPTION
**Pull request by oshezaf**
Includes:
  - Normalization parsers
  - Converted content

### CODE REVIEW

**Comment by oshezaf** [comment]
Resolved "notebooks" merge issue and passed tests

File: Hunting Queries/ASimProcess/imProcess_Suspicious_enumeration_using_adfind.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  - NOBELIUM
+query: |
+  let lookupwindow = 2m;
+  let threshold = 3; //number of commandlines in the set below
+  let DCADFSServersList = dynamic (["DCServer01", "DCServer02", "ADFSServer01"]); // Enter a reference list of hostnames for your DC/ADFS servers
+  let tokens = dynamic(["objectcategory","domainlist","dcmodes","adinfo","trustdmp","computers_pwdnotreqd","Domain Admins", "objectcategory=person", "objectcategory=computer", "objectcategory=*"]);
+  imProcessCreate
+  //| where Computer in (DCADFSServersList) // Uncomment to limit it to your DC/ADFS servers list if specified above or any pattern in hostnames (startswith, matches regex, etc).
+  | where CommandLine has_any (tokens)
+  | where CommandLine matches regex "(.*)>(.*)"
```
Comment (YaronFruchtmann) [inline Hunting Queries/ASimProcess/imProcess_Suspicious_enumeration_using_adfind.yaml:36]: maybe "where CommandLine contains '>' "

**Review by shainw — CHANGES_REQUESTED** [review]
Other suggestions inline - The only thing that "has" to be changed is removing the submodule updates for Notebooks.  You can get this out of your way by right clicking Notebooks (submodule) in VS during the commit and choosing update:
submoduleUpdate

File: Parsers/ASimProcess/ARM/ProcessEventGeneric/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Source Agnostic ASIM ProcessEvent Normalization Parser
+
+This template deploys the Source Agnostic ASIM ProcessEvent schema parser. The parser is a part of the Azure Sentinel Information Model.
```
Comment (shainw) [inline Parsers/ASimProcess/ARM/ProcessEventGeneric/README.md:3]: This is the same as the Empty parser information, shouldn't we make it clear this is for Generic?

**Review by shainw — CHANGES_REQUESTED** [review]

**Comment by oshezaf** [comment]
All issues fixed

**Pull request merged**